### PR TITLE
done with landing page of component library

### PR DESCRIPTION
**Pull request by dev-prabhat**

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
Great work. All good

File: index.html [hunk 1]
```diff
@@ -14,40 +20,63 @@
...
+              it in your web application, gives your UI with Superpower of CSS
+            </p>
+            <a class="btn btn-primary btn-link" href="#snippet"
+              >Getting Start</a
+            >
+            <a class="btn btn-secondary btn-link" href="/">Documentation</a>
+          </section>
+
+          <section class="hero-img">
+            <img
```
Comment (humzahasan) [inline index.html:49]: Image is too big.

File: index.html [hunk 2]
```diff
@@ -14,40 +20,63 @@
...
+          <section class="install-instruction">
+            <h2 class="title head-2 margin-md">Installation</h2>
+            <p class="description margin-md text-4">
+              Copy and paste the code given in the iframe in your head tag of
+              your html document , and implement classes as per defined to give
+              Superpower to your components
+            </p>
+          </section>
+          <div id="snippet" class="code-snippet">
+            <iframe
```
Comment (humzahasan) [inline index.html:67]: Make iframe bigger, since that's your CTA. Image size can be reduced.

File: index.html [hunk 3]
```diff
@@ -14,40 +20,63 @@
...
-              border: 0;
-              transform: scale(1);
-              overflow: hidden;
-            "
-            sandbox="allow-scripts allow-same-origin"
-          >
-          </iframe>
-        </div>
+      <main class="container main-container">
+        <section class="hero-section">
```
Comment (dcs-soni) [inline index.html:33]: Everything looks good to me. Just a suggestion - For a better understanding of sections, the class names of the hero sections can be named as
- hero-section
   - hero-info
      - hero-title
      - hero-description

**Review by dcs-soni — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (dev-prabhat) [reply index.html:49]: Done,I have decreased width by 20%

**Review by humzahasan — APPROVED** [review]
Looks Good Prabaht!

**Pull request merged**